app.js

```diff
@@ -0,0 +1,58 @@
+if (localStorage.getItem('Added Books') == null) {
+  localStorage.setItem('Added Books', JSON.stringify([]));
+}
+
+const storeData = JSON.parse(localStorage.getItem('Added Books'));
+
+function updateData() {
+  localStorage.setItem('Added Books', JSON.stringify(storeData));
+}
+
+function createBook(arr) {
+  let books = '';
+  for (let i = 0; i < arr.length; i += 1) {
+    books += `
+      <p>${arr[i].title}</p>
+      <p>${arr[i].author}</p>
+      <button onclick = "removeBook(${i})">Remove</button>
+      <hr/>
+    `;
+  }
+  return books;
+}
+
+function displayBooks() {
+  const listOfBooks = document.querySelector('.container');
+  listOfBooks.innerHTML = `
+    <ul>
+      ${createBook(storeData)}
+    </ul>
+  `;
+}
+
+function addNewData(bookTitle, bookAuthor) {
+  const Book = {
+    title: bookTitle,
+    author: bookAuthor,
+  };
+  storeData.push(Book);
+  updateData();
+  displayBooks();
+}
+
+const bookForm = document.querySelector('form');
+bookForm.addEventListener('submit', (e) => {
+  const title = document.querySelector('.title');
+  const author = document.querySelector('.author');
+  e.preventDefault();
+  addNewData(title.value, author.value);
+});
+
+function removeBook(i) {
+  storeData.splice(i);
```

Comment:
- [ ] It seems like you're missing one more argument for the `splice` method as it's not working properly check the video below:
  ![firefox_btxEpTkvtm](https://user-images.githubusercontent.com/104968236/234036203-b438b8a5-3f51-47ea-b9d1-2324ac76e6ce.gif)
- As you can see from the above video only 1 book should've been removed so, please make sure to fix that. 
- You can find more info about the `splice()` method [here](https://www.w3schools.com/jsref/jsref_splice.asp).